test/loader.test.js

```diff
@@ -0,0 +1,29 @@
+const { spawnPromisified, fixturesPath } = require("./util/util.js");
+const { test } = require("node:test");
+const { match, strictEqual } = require("node:assert");
+
+test("should work as a loader", async () => {
+	const result = await spawnPromisified(process.execPath, [
+		"--experimental-strip-types",
```

Comment:
Can we get rid of the `--experimental-strip-types` flag if amaro is working as an independent loader? It is only necessary for the `format` detection, which can be resolved with a resolve hook.

Reply:
Yeah, I think it will need a bunch of replication of the resolution done in core. This can a follow-up since without this flag could make it easier to use.